sections/language_model.tex

```diff
@@ -0,0 +1,21 @@
+\section{Language model}\label{sec:lm}
```

Comment:
Seems like we're missing an intro saying 'now we will go over some of the theory behind the methods that we will be using.'
maybe recombine LDA PR and LM into one section and have them be subsections.